### PR TITLE
SC-3185 - Adjust scripts based on new agent commands

### PR DESCRIPTION
**Pull request by hakman**

### CODE REVIEW

File: manifests/configure.pp [hunk 1]
```diff
@@ -4,193 +4,133 @@
...
+#   Monitoring Token unique to each application
 #
-# [*spm_type*]
-#   SPM application type (apache, haproxy, javaagent, ...)
+# [*infra_token*]
+#   Infra Token unique to each account
 #
-# [*apache_stats_url*]
-#   Apache server status URL provided by mod_status
+# [*app_type*]
```
Comment (sivasamyk) [inline manifests/configure.pp]: This should be `agent_type` I think

File: manifests/configure.pp [hunk 2]
```diff
@@ -4,193 +4,133 @@
...
+
+    if !has_key($args, 'app_subtype') {
+      $command = "/opt/spm/bin/setup-sematext \\
+        --monitoring-token ${ monitoring_token } \\
+        --infra-token ${ infra_token } \\
+        --agent-type ${ agent_type } \\
+        --app-type ${ app_type } \\
+        ${extra_args}"
+
+      $config = "/opt/spm/spm-monitor/conf/spm-monitor--config-${ monitoring_token }-${ app_name }.properties"
```
Comment (sivasamyk) [inline manifests/configure.pp]: we check for `app_subtype` in the condition above and use `app_name` in config.

Comment (hakman) [reply manifests/configure.pp]: we check for **not** having 'app_subtype' above.

File: manifests/configure.pp (hunk 1 above)
Comment (hakman) [reply manifests/configure.pp]: true

**Pull request merged**